custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_v2.json

```diff
@@ -1377,6 +1377,43 @@
               }
             },
             "property_value": 183
+          },
+          {
+            "type": "or",
```

Comment:
@calellowitz I'm pretty sure that the rules are:
"valid" = registered_status != not_registered and migrated != yes

Basically as soon as you're migrated OR as soon as you're marked as "not registered" you're no longer considered a valid beneficiary.  Either should exclude them.